app/views/granules/prepare_script.html.erb

```diff
@@ -43,7 +43,12 @@
 <body>
 <div id="errors"></div>
 <div id="loading">Preparing download script. Fetching downloadable granule links...</div>
-<div id="download-description" style="display: none;">Script preparation completed. Please click the button to download the script. <a href="#" id="download-button">Download Script File</a></div>
+<div id="download-description" style="display: none;">
+  <h2>How to use this script</h2>
+  <p>Linux: You must first make the script an executable by running the line 'chmod 777 download.sh' from the command line.  After that is complete, the file can be executed by typing './download.sh'.</p>
```

Comment:
Let's add a line to the effect of "For a detailed walk through of this process, read this How To guide." and make that link (and open in a new window/tab) to this wiki page - https://wiki.earthdata.nasa.gov/display/EDSC/How+To%3A+Use+the+Download+Access+Script